### PR TITLE
Added auto-topup settings

### PR DESCRIPTION
**Pull request by freeduerinckx**
Modelled the end points to read and update the auto-topup settings.
Removed the old existing once that weren't clean up properly.

Related to: https://vikingco.atlassian.net/browse/UWA-778

### CODE REVIEW

**Comment by freeduerinckx** [comment]
@Linsy can you check if this is ok by you. @Kjoep can you check if this is reasonable if you want to use this with Jim in the future?

File: definitions/subscriptions/auto_topup_settings.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    description: A list of trigger properties that might trigger a topup
+    type: array
+    items:
+      $ref: '#/definitions/AutoTopupTrigger'
+  mandate_id:
+    type: string
+    description: The id of the mandate that will be used to pay the automatic topup when it takes place.
+  product_id:
+    type: string
+    description: The if of the product that will be used to topup when an automatic topup takes place.
```
Comment (freeduerinckx) [inline definitions/subscriptions/auto_topup_settings.yaml]: Two settings 
- checkbox to allow sms auto topup
- use viking points first

**Comment by Kjoep** [comment]
It seems good to me - and it's compatible with the previous model.

I do have to ask my usual question -- what is the motivation for changing this?  The model that was there seems to do the same thing and does not seem worse to me.  And since the frontend for that already works (even if it is hidden for the moment) it seems a bit silly to change it.

Of course if you have a good reason for this I'm fine.

**Comment by freeduerinckx** [comment]
@Kjoep the previous model was less future proof. Any change (addition of deletion of a trigger) would require code changes. 
E.g. adding number_of_free_smses as trigger, it would require code change everywhere now it can be done in Comarch and it should require any change on our side (uwa and frontends).

**Comment by Kjoep** [comment]
Are you implying that we're channeling through the 'trigger' codes from comarch directly?  Shoudn't uwa act as a buffer there? If not, we'd need to adapt uwa-model (which we should stay master of) to some volatile config on comarch.  That seems like a weird coupling.

As for the frontends, they would need to be adapted when a trigger is removed/added anyway.  The only difference is whether they are coupled to uwa or to comarch.  (I prefer uwa, myself -- it's more stable)

If it fits your planning, lets continue this IRL :)

**Comment by freeduerinckx** [comment]
@Kjoep I send a meeting request for tomorrow as you are working form how today.

**Comment by freeduerinckx** [comment]
@Kjoep I pushed a fix up with the change we had discussed this morning

**Pull request merged**